boards/silabs/radio_boards/slwrb4180b/slwrb4180b.yaml

```diff
@@ -0,0 +1,21 @@
+identifier: slwrb4180b
+name: EFR32xG21 2.4 GHz 20 dBm Radio Board (SLWRB4180B)
+type: mcu
+arch: arm
+ram: 96
+flash: 1024
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - counter
+  - gpio
+  - nvs
+  - uart
+  - watchdog
+testing:
+  ignore_tags:
+    - net
+    - bluetooth
```

Comment:
Is something breaking if you don't list this here? I'd have imagined that it's enough to omit `bluetooth` from the `supported` list rather than to have to explicitly exclude it through `ignore_tags`. Same for `net`. That said, I realize this is likely just copy-pasted stuff, but it would be nice to sometimes stop and think if it all makes any sense or not :)